apps/system/lockscreen/js/charging.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+(function(exports) {
+
+/**
+ * Creates an object used for refreshing the charging string UI element.
+ * @class  ChargingStatus
+ */
+function ChargingStatus() {
+  this.refreshCB = null;
+}
+
+ChargingStatus.prototype.start = function cs_start() {
+  this.refresh();
+
+  if (!this.refreshCB) {
```

Comment:
I think you can put `this.refresh = this.refresh.bind(this)` constructor. If the method would not be bound to other APIs it's OK to overwrite it in constructor, and we can get rid of the suffix 'CB'.